### PR TITLE
Reduce size of extra content in error output truncation test

### PR DESCRIPTION
**Pull request by helderco**
First attempt at reducing the performance impact during tests from:
- https://github.com/dagger/dagger/pull/5164

Reported in:
- https://github.com/dagger/dagger/pull/5163#issuecomment-1563075893

### CODE REVIEW

**Comment by helderco** [comment]
Screenshot 2023-05-25 at 16 00 25

File: core/integration/container_test.go [hunk 1]
```diff
@@ -3000,26 +3000,23 @@ func TestContainerExecError(t *testing.T) {
 	})
 
 	t.Run("truncates output past a maximum size", func(t *testing.T) {
-		// fill a byte buffer with a string that is twice the size of the max output
+		// fill a byte buffer with a string that is slightly over the size of the max output
 		// size, then base64 encode it
 		var stdoutBuf bytes.Buffer
-		for i := 0; i < 2*core.MaxExecErrorOutputBytes; i++ {
+		for i := 0; i < core.MaxExecErrorOutputBytes+50; i++ {
```
Comment (gerhard) [inline core/integration/container_test.go:3006]: Nit: could we explain in a follow-up PR what that `50` is? I always like to capture the meaning of values in a descriptive variable name, comments are easy to miss.

This is an important change, going to merge it straight away.

**Review by gerhard — APPROVED** [review]
👍 🚀

File: core/integration/container_test.go (hunk 1 above)
Comment (helderco) [reply core/integration/container_test.go:3006]: It's explained in the comment above it:

Screenshot 2023-05-25 at 17 46 09

**Pull request merged**